<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(0,0,0);
}

QMenuBar{
background-color: rgb(0,0,0);
}
QLabel{
	color: #FFF;
	padding-left: 20px;
	padding-right: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 10px;
}

QGroupBox::title {
    color: white; /* Change this to the color you want */
}
</string>
    </property>
    <property name="title">
     <string>Low Stock Items</string>
    </property>
    <widget class="QTableWidget" name="lowStockList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>441</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	background-color: rgb(0,0,0);
	gridline-color: rgb(255, 255, 255);
	color: rgb(255,255,255);
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
	selection-background-color: rgb(46, 46, 62);
}</string>
     </property>
     <column>
      <property name="text">
       <string>ItemID</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Colour</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Quantity Left</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>These items are low in stock. Place a Purchase Order for them</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 10px;
}

QGroupBox::title {
    color: white; /* Change this to the color you want */
}
</string>
    </property>
    <property name="title">
     <string>Top Selling Items</string>
    </property>
    <widget class="QTableWidget" name="topSellingList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	background-color: rgb(0,0,0);
	gridline-color: rgb(255, 255, 255);
	color: rgb(255,255,255);
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
	selection-background-color: rgb(46, 46, 62);
}</string>
     </property>
     <column>
      <property name="text">
       <string>ItemID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ttl Quantity Sold</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
